### PR TITLE
Added firmware fingerprint IONIQ 2020 EV

### PR DESCRIPTION
**Pull request by kovat74**
<!--- ***** Template: Fingerprint *****

**Car**
Hyundai IONIQ 2020 EV

**Route**
555e839edbae7c0c|2024-01-18--08-54-40--0

The Car really is IONIQ 2021, but put the firmwares under IONIQ 2020 in fingerprints.py
Worked fine.

### CODE REVIEW

**Comment by sshane** [comment]
| Dongle, platform, VIN                                                                                                                  | Name from VIN 🆚 CarInfo   | Segments                  | Bad seg tags   | Good seg tags                                           |
|:---------------------------------------------------------------------------------------------------------------------------------------|:---------------------------|:--------------------------|:---------------|:--------------------------------------------------------|
| [555e839edbae7c0c](https://useradmin.comma.ai/?onebox=555e839edbae7c0c)<br><sub>HYUNDAI IONIQ ELECTRIC 2020<br>000000000........</sub> | None 🆚<br>None            | 16 routes, 162 segments   |                | - all lat active: 65<br>- no input all lat active: 43   |

**Comment by kovat74** [comment]
555e839edbae7c0c|2024-01-24--06-12-21--0

File: selfdrive/car/hyundai/interface.py [hunk 1]
```diff
@@ -137,6 +137,11 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.wheelbase = 2.97
       ret.steerRatio = 14.26
       ret.tireStiffnessFactor = 0.65
+      #ret.lateralTuning.init('indi')
+      #ret.lateralTuning.indi.innerLoopGain = 3.0
+      #ret.lateralTuning.indi.outerLoopGain = 2.0
+      #ret.lateralTuning.indi.timeConstant = 1.0
+      #ret.lateralTuning.indi.actuatorEffectiveness = 1.0
```
Comment (sshane) [inline selfdrive/car/hyundai/interface.py:144]:
```suggestion
```

like this

**Comment by jnewb1** [comment]
<details open><summary>View table</summary><br>

| Dongle, platform, VIN                                                                                                                  | Name from VIN 🆚 CarInfo   | Segments                | Bad seg tags   | Good seg tags                                           |
|:---------------------------------------------------------------------------------------------------------------------------------------|:---------------------------|:------------------------|:---------------|:--------------------------------------------------------|
| [555e839edbae7c0c](https://useradmin.comma.ai/?onebox=555e839edbae7c0c)<br><sub>HYUNDAI IONIQ ELECTRIC 2020<br>000000000........</sub> | None 🆚<br>None            | 28 routes, 374 segments |                | - all lat active: 164<br>- no input all lat active: 103 |
</details>

**Pull request merged**